### ISSUE TITLE
Minor writing typo

### ISSUE THREAD

**Issue by LeonardoRM** [daumann/ECMAScript-new-features-list#17]
In this file, there's a minor error in the word "**enumerable**", twice in the following paragraph.

File: [ECMAScript-new-features-list/ES2017.MD](https://github.com/daumann/ECMAScript-new-features-list/blob/master/ES2017.MD#objectentries)

> ### `Object.entries()`
> 
> `Object.entries` gives us the ability to get an object's **enurmerable** property pairs by returning an array of any given object's own **eumberable** properties. /ie: [key, value] pairs. Note that the order is the same as provided by the `for...in` loop.

**Comment by mbean0302** [comment]
Good catch. I just recently forked this repo and submitted a pull request for fixing the typos.
